sqlgen/reflect.go

```diff
@@ -455,32 +455,48 @@ func makeWhere(table *Table, filter Filter) (*SimpleWhere, error) {
 	}, nil
 }
 
-// makeSelect builds a new SelectQuery for table with filter
-func (s *Schema) makeSelect(typ reflect.Type, filter Filter, options *SelectOptions) (*SelectQuery, error) {
-	table, err := s.get(typ)
-	if err != nil {
-		return nil, err
-	}
+type BaseSelectQuery struct {
```

Comment:
I agree, but I don't want to tackle cleaning up the SelectQuery / BaseQuery / etc. mess in this PR. All those structs ought to be internal implementation details, but they won't be as long as livesql lives in its own package. I plan to move livesql into sqlgen at some point, make all those structs private, and rename them to something better.